src/crud.js

```diff
@@ -71,13 +72,28 @@ export default (server, model, { prefix, defaultConfig: config, models: permissi
     return params;
   }, {});
 
-  const validAssociations = modelAssociations.length
-    ? joi.string().valid(...modelAssociations)
-    : joi.valid(null);
+  const modelsHasAssociations = modelAssociations && modelAssociations.length;
+  const validAssociationsString = modelsHasAssociations
+      ? joi.string().valid(...modelAssociations)
+      : joi.valid(null);
+  const validAssociationsObject = modelsHasAssociations
+      ? joi.object().keys({
```

Comment:
yes maybe. especially if you can express what the possibility of vulnerability that might happen to this include feature. i have tried something like this to mess the JSON include parameter:
```
/cities?include={ model: "Player" } // it should be: /cities?include={ "model": "Player" }
/cities?include=model: "Player" } // it should be: /cities?include={ "model": "Player" }
```
this is the both result:
![screen shot 2016-11-11 at 9 17 57 am](https://cloud.githubusercontent.com/assets/6038766/20202161/cc39e6ac-a7ef-11e6-93cf-4936799a51fd.png)